graphdatascience/utils/util_remote_proc_runner.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, List
+
+from ..error.illegal_attr_checker import IllegalAttrChecker
+from ..error.uncallable_namespace import UncallableNamespace
+from graphdatascience.error.cypher_warning_handler import (
+    filter_id_func_deprecation_warning,
+)
+from graphdatascience.utils.util_node_property_func_runner import NodePropertyFuncRunner
+
+
+class UtilRemoteProcRunner(UncallableNamespace, IllegalAttrChecker):
+    @filter_id_func_deprecation_warning()
+    def asNode(self, node_id: int) -> Any:
+        """
+        Get a node from a node id.
+
+        Args:
+            node_id: The id of the node to get.
+
+        Returns:
+            The node with the given id.
+
+        """
+        query = "MATCH (n) WHERE id(n) = $nodeId RETURN n"
+        params = {"nodeId": node_id}
+
+        return self._query_runner.run_cypher(query=query, params=params).squeeze()
+
+    @filter_id_func_deprecation_warning()
+    def asNodes(self, node_ids: List[int]) -> List[Any]:
+        """
+        Get a list of nodes from a list of node ids.
+
+        Args:
+            node_ids: The ids of the nodes to get.
+
+        Returns:
+            The nodes with the given ids.
+
+        """
+        query = "MATCH (n) WHERE id(n) IN $nodeIds RETURN collect(n)"
+        params = {"nodeIds": node_ids}
+
+        return self._query_runner.run_cypher(query=query, params=params).squeeze()  # type: ignore
```

Comment:
Why are these only in the remote space?

Reply:
They are also on the `util_proc_runner` but there we use the existing GDS functions such as `gds.util.asNode`